exchanges/gateio/gateio_websocket.go

```diff
@@ -126,7 +126,7 @@ func (e *Exchange) websocketLogin(ctx context.Context, conn websocket.Connection
 	signature := hex.EncodeToString(mac.Sum(nil))
 
 	payload := WebsocketPayload{
-		RequestID: strconv.FormatInt(conn.GenerateMessageID(false), 10),
+		RequestID: e.MessageID(),
```

Comment:
```suggestion
		RequestID: e.MessageSequence(),
```

Reply:
This was deliberate.
We don't log in very often so I've chosen to use a unique id.